PS8lite.Rmd

```diff
@@ -187,7 +190,24 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+#MVP: model_rf <- randomForest(form = model_formula, data = training, mtry = 2, ntree = 100)
 
+#Defining cv settings
+fit_control <- trainControl(method = "cv", number = 10)
+
+#Search grid for tuning parameters (mtry)
+mtry <- 2:6
+tunegrid <- expand.grid(.mtry = mtry)
+
+#Fit/train model on training data
+model_rf <- caret::train(form = model_formula, 
+                               data = training, 
+                               method = "rf", 
+                               metric = "RMSE", 
+                               trControl = fit_control, 
+                               tuneGrid = tunegrid)
+
+model_rf #See which mtry value gets used
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.